webclient/static/js/dz.js

```diff
@@ -201,9 +201,11 @@ window.onload = function () {
                 } else {
                     show_upload_successful_message(file, result, message);
                 }
-                var submitButton = $(".btn-disable-when-running-upload");
-                submitButton.prop('disabled', false); 
-                submitButton.removeClass("disabled"); 
+                if (this.files.every(file => file.status === 'success')) {
```

Comment:
@jhavrlant A co když to success nebude? Je to nějak ošetřeno (např. chyba připojení apod.)?

Reply:
Jak jsem to testoval, tak pokud byl se souborem nějaký problém, tak ho stejně odebral Drobzone nebo ho odebere uživatel a znovu nahraje. Zde je file status ve smyslu, jestli se nahrává nebo je hotovo.